src/utils/isCollected.ts

```diff
@@ -0,0 +1,5 @@
+const isCollected = (cards: PokemonCardObject[], id: string) => {
```

Comment:
When creating function, I almost always just accept an object. That way, order doesn't matter, you can omit things easily, and it's clear what each field is:

```suggestion
const isCollected = ({ cards, id }: isCollectedArgs) => {
```